examples/gfx_advanced/src/main.rs

```diff
@@ -517,14 +519,37 @@ fn update_inputs(events_loop: &mut glutin::EventsLoop, scene: &mut SceneParams)
     use glutin::Event;
     use glutin::VirtualKeyCode;
     use glutin::ElementState::Pressed;
+    use glutin::WindowEvent;
 
     events_loop.poll_events(|event| {
         match event {
             Event::WindowEvent {event: glutin::WindowEvent::Closed, ..} => {
                 println!("Window Closed!");
                 status = false;
-            },
-            Event::WindowEvent {event: glutin::WindowEvent::KeyboardInput {input: glutin::KeyboardInput {state: Pressed, virtual_keycode: Some(key), ..}, ..}, ..} => {
+            },           
+            Event::WindowEvent {
+                event: WindowEvent::MouseInput {
+                    state: ElementState::Pressed, button: glutin::MouseButton::Left,
```

Comment:
Almost there. Travis ci appears to complain about the `ElementState` symbol not being imported. Since `ElementState::Pressed` is imported you can just write `Pressed` here to get it to compile.